www/js/recommendations.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+angular.module('emission.main.recommendations',['emission.services', 'emission.plugin.logger',
+                'emission.survey.launch',
+                'ngSanitize', 'ngAnimate',
+                'emission.splash.referral', 'angularLocalStorage',
+                'ng-walkthrough', 'nzTour'])
+
+.controller('RecommendationsCtrl', function(CommHelper, $state, $ionicLoading, $scope, $rootScope, $ionicModal, nzTour,
+                                $window, $http, $ionicPopup, $timeout, storage, ReferralHandler, ReferHelper, Logger, $cordovaInAppBrowser, SurveyLaunch) {
+    $scope.cSF = 1000;
+    var map = document.getElementById('mapModal');
```

Comment:
nit: convert to angular style. Look at the list and detail screen for examples.
can defer.